pkg/devicemapper/devmapper.go

```diff
@@ -264,11 +264,6 @@ func UdevWait(cookie *uint) error {
 	return nil
 }
 
-// LogInitVerbose is an interface to initialize the verbose logger for the device mapper library.
-func LogInitVerbose(level int) {
```

Comment:
`LogInitVerbose` is meaningless if you are using callbacks. It only has meaning for the default logger, which we don't use because we need callbacks to get errors.